data/xml/Q17.xml

```diff
@@ -216,102 +232,124 @@
-    <href>https://transacl.org/ojs/index.php/tacl/article/download/1064/242</href>
+    <month>December</month>
+    <doi>10.1162/tacl_a_00058</doi>
+    <abstract>Our goal is to construct a domain-targeted, high precision knowledge base (KB), containing general (subject,predicate,object) statements about the world, in support of a downstream question-answering (QA) application. Despite recent advances in information extraction (IE) techniques, no suitable resource for our task already exists; existing resources are either too noisy, too named-entity centric, or too incomplete, and typically have not been constructed with a clear scope or purpose. To address these, we have created a domain-targeted, high precision knowledge extraction pipeline, leveraging Open IE, crowdsourcing, and a novel canonical schema learning algorithm (called CASI), that produces high precision knowledge targeted to a particular domain - in our case, elementary science. To measure the KB’s coverage of the target domain’s knowledge (its “comprehensiveness” with respect to science) we measure recall with respect to an independent corpus of domain text, and show that our pipeline produces output with over 80% precision and 23% recall with respect to that target, a substantially higher coverage of tuple-expressible science knowledge than other comparable resources. We have made the KB publicly available.</abstract>
     <pages>233–246</pages>
-    <url>http://www.aclweb.org/anthology/Q17-1017</url>
+    <url>https://www.aclweb.org/anthology/Q17-1017</url>
   </paper>
 
   <paper id="1018">
     <title>Sparse Coding of Neural Word Embeddings for Multilingual Sequence Labeling</title>
-    <author><first>Gábor</first><last>Berend</last></author>
+    <author><first>Gáabor</first><last>Berend</last></author>
     <year>2017</year>
+    <month>December</month>
+    <doi>10.1162/tacl_a_00059</doi>
     <abstract>In this paper we propose and carefully evaluate a sequence labeling framework which solely utilizes sparse indicator features derived from dense distributed word representations. The proposed model obtains (near) state-of-the art performance for both part-of-speech tagging and named entity recognition for a variety of languages. Our model relies only on a few thousand sparse coding-derived features, without applying any modification of the word representations employed for the different tasks. The proposed model has favorable generalization properties as it retains over 89.8% of its average POS tagging accuracy when trained at 1.2% of the total available training data, i.e. 150 sentences per language.</abstract>
-    <href>https://transacl.org/ojs/index.php/tacl/article/download/1063/241</href>
     <pages>247–261</pages>
-    <url>http://www.aclweb.org/anthology/Q17-1018</url>
-    <attachment type="presentation">Q17-1018.Presentation.pdf</attachment>
```

Comment:
It looks like we are losing attachments with this merge.

Reply:
Fixed in last night's version